optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -1307,6 +1351,187 @@ def get_text_embeddings(self, input_ids, **kwargs):
         return super().get_text_embeddings(for_inputs_embeds_ids, **kwargs)
 
 
+class _OVLlavaNextVideoForCausalLM(_OVLlavaNextForCausalLM):
+    additional_parts = ["vision_resampler", "multi_modal_projector"]
+    export_feature = "video-text-to-text"
```

Comment:
As I understand this model also supports image inputs instead of video inputs, however the export feature is still `"video-text-to-text"`. Can we assume in the future that all models with `export_feature="video-text-to-text"` also support image inputs?

Reply:
yes, it supports. this export_featere does not affect export and can be removed, because OVModelForVisualCausalLM has image-text-to-text